dspace-api/src/main/java/org/dspace/discovery/DiscoverResult.java

```diff
@@ -28,8 +28,11 @@
 
     private long totalSearchResults;
     private int start;
-    private List<DSpaceObject> dspaceObjects;
+    private List<BrowsableObject> dspaceObjects;
     private Map<String, List<FacetResult>> facetResults;
+    private Map<String, List<FacetResult>> facetQueryResults;
```

Comment:
Variable isn't used, neither getters nor setters are called. Can you remove it ?

Reply:
removed, see https://github.com/4Science/DSpace/pull/65/commits/cc11f4955677eaedfae70c54eb08e940bb6067fe